packages/pg/lib/client.js

```diff
@@ -848,7 +849,7 @@ class Client extends EventEmitter {
   _handleErrorWhileConnecting(err) {
     if (this._connectionError) {
       // TODO(bmc): this is swallowing errors - we shouldn't do this
-      if (this.connectionParameters.loadBalance) {
+      if (this.connectionParameters.loadBalance || Client.controlClient === undefined) {
```

Comment:
This may change the driver behaviour when load-balance is disabled. That may be ok since we do not encourage to disable load-balance when our driver is being used.